### DISCUSSION TITLE
How can I convert or download the output that I get using the command of objects from FastHerbie as text or csv file?

### DISCUSSION THREAD

**Discussion by shihab-saran** [blaylockbk/Herbie#372] [Q&A]
I was trying to look a set of available HRRR files from a certain range of date to make available data list for my research. I got the output but I didn’t know how I can save them or download them in my local computer (windows) as a txt or csv or any readable file?

**Comment by williamhobbs** [comment]
Can you provide a sample of your code? And/or more details about what you are trying to do? For example, do you want a timeseries of a single weather variable for a single location (e.g., 24-hour ahead surface temperature forecasts for Atlanta)? 

If you want gridded data (e.g., the whole HRRR footprint) over multiple time intervals, then grib or netcdf files are probably your best option, but you might not consider those as "readable" as txt or csv.
